test/lib/ansible_test/_internal/docker_util.py

```diff
@@ -275,7 +301,7 @@ def docker_version(args):
     return json.loads(stdout)
 
 
-def docker_command(args, cmd, capture=False, stdin=None, stdout=None, always=False):
+def docker_command(args, cmd, capture=False, stdin=None, stdout=None, always=False, data=None):
```

Comment:
Add `data` to the type hints in the docstring.

Reply:
Done.